### ISSUE TITLE
No working with latest oauth2client==3.0.0

### ISSUE THREAD

**Issue by xnone** [googleads/googleads-python-lib#136]
`'oauth2client>=1.5.2,<2.0.0'` is required but we need some function from latest oauth2client. 
Thus it stops the googleads. 
Is it possible to solve this dependency issue?

**Comment by msaniscalchi** [comment]
This is a duplicate of Issue #110.

I can say that we're much closer to a resolution in comparison to my last update, but we have some remaining prep work on our end before we can push the fix here.
